### ISSUE TITLE
NTR: Imidazolelactic acid high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8846]
New term request Imidazolelactic acid high in urine (elthat_111113034817):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000531
- label: Elevated urinary imidazolelactic acid level
- parents: [Elevated urinary monocarboxylic acid level (HP:0033303)](https://hpo.jax.org/app/browse/term/HP:0033303)
- pmid: PMID:4645251
- def: The amount of imidazolelactic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: Urinary excretion of imidazolelactic acid may be increased in persons with folic acid or vitamin B12-deficiency.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8846
- synonyms:

**Comment by MickeySegal** [comment]
OK.  Deleting.
